cmd/siac/main.go

```diff
@@ -75,7 +119,14 @@ func main() {
 	walletCmd := flagg.New("wallet", walletUsage)
 	walletBalanceCmd := flagg.New("balance", walletBalanceUsage)
 	exactBalance := walletBalanceCmd.Bool("exact", false, "print balance in Hastings")
-
+	walletAddressCmd := flagg.New("address", addressUsage)
+	walletAddressesCmd := flagg.New("addresses", addressesUsage)
+	walletTransactionsCmd := flagg.New("transactions", transactionsUsage)
+	transactionID := walletTransactionsCmd.String("id", "", "print only a specific transaction by its ID")
```

Comment:
seems kinda unnecessary; you can `grep` for a particular ID. Maybe instead we could have a separate `transaction` command that nicely formats all of the details of a particular transaction.